modules/ROOT/pages/storage.adoc

```diff
@@ -4,6 +4,13 @@ Fedora CoreOS ships with a simple default storage layout: the root partition is
 
 Below, we provide examples of various ways you can customize this.
 
+[NOTE]
+====
+*Fedora CoreOS requires the root filesystem to be at least 8 GiB.* For practical purposes, disk images ship with a smaller root filesystem on some platforms, which by default automatically expands to fill all available disk space. If you add additional partitions after the root filesystem, you must make sure to explicitly resize the root partition as shown below so that it is at least 8 GiB.
```

Comment:
That's not quite what `for practical purposes` means.  Also, `which` could refer to `platforms` on an initial parse.  How about:

```suggestion
*Fedora CoreOS requires the root filesystem to be at least 8 GiB.* For practical reasons, disk images for some platforms ship with a smaller root filesystem, which by default automatically expands to fill all available disk space. If you add additional partitions after the root filesystem, you must make sure to explicitly resize the root partition as shown below so that it is at least 8 GiB.
```

Reply:
Thanks, tweaked!